src/CleanUpModelsCommand.php

```diff
@@ -64,6 +64,9 @@ protected function cleanUp(Collection $cleanableModels)
 
             $numberOfDeletedRecords = $class::cleanUp($class::query())->delete();
 
+            if($numberOfDeletedRecords > 0)
```

Comment:
Please use curly braces for an if statement. Checking on `> 0` is not necessary in this case.